FinnHub.Core/FinnHubClient.cs

```diff
@@ -56,13 +59,18 @@ public async Task<CompanyInfo> GetCompanyInfoAsync(string ticker)
                 {
                     using (var response = await client.SendAsync(request))
                     {
-                        if (response.IsSuccessStatusCode)
+                        if (!response.IsSuccessStatusCode) return null;
```

Comment:
We should really be throwing an exception with the response message here instead of just returning null.